node/grpc/utils.go

```diff
@@ -54,17 +54,27 @@ func GetBlobMessages(in *pb.StoreChunksRequest, numWorkers int) ([]*core.BlobMes
 				return
 			}
 
+			format := node.GetBundleEncodingFormat(blob)
 			bundles := make(map[core.QuorumID]core.Bundle, len(blob.GetBundles()))
-			for j, chunks := range blob.GetBundles() {
+			for j, bundle := range blob.GetBundles() {
 				quorumID := blob.GetHeader().GetQuorumHeaders()[j].GetQuorumId()
-				bundles[uint8(quorumID)] = make([]*encoding.Frame, len(chunks.GetChunks()))
-				for k, data := range chunks.GetChunks() {
-					chunk, err := new(encoding.Frame).Deserialize(data)
+				if format == core.GnarkBundleEncodingFormat {
+					bundleMsg, err := new(core.Bundle).Deserialize(bundle.GetBundle())
```

Comment:
This deserialization will be 3.6x faster (https://github.com/Layr-Labs/eigenda/pull/624)